libs/data-access/gn4/src/openapi/model/metadataResource.api.model.ts

```diff
@@ -19,7 +19,7 @@ export interface MetadataResourceApiModel {
   metadataResourceExternalManagementProperties?: MetadataResourceExternalManagementPropertiesApiModel
   lastModification?: string
   version?: string
-  url?: string
+  url?: URL
```

Comment:
these files are auto-generated; this change will be list if we regenerate the API client, so I would avoid doing this